youtubesearchpython/extras.py

```diff
@@ -250,7 +255,9 @@ def get(videoLink: str, mode: int = ResultMode.dict, timeout: int = None) -> Uni
                     ]
                 }
         '''
-        return Video(videoLink, None, mode, timeout).result
+        vc = VideoCore(videoLink, None, mode, timeout)
+        vc.sync_create()
```

Comment:
Somewhere we're using camel cased attributes like `_nextAsync` & here it is snake cased.

Maybe following snake case in attributes, function-names & variable names is more common in Python. Although I'm fine with everything. For me, if it works, it's fine.

Or is this yet another Python feature? EDIT: I see declaration.

Reply:
This is quite a big rewrite. I'll probably do it in the future.